views/register.html

```diff
@@ -235,11 +236,10 @@ <h3 class="header-red">
   function manageSubscriptionStatus(subscription) {
     const { latest_invoice } = subscription;
     const { payment_intent } = latest_invoice;
-
     if (payment_intent) {
       /* Do NOT share or embed your client_secret anywhere */
       const { client_secret, status } = payment_intent;
-      if (status === "requires_action" || status === "requires_payment_method") {
+      if (status === "requires_action" || status === "requires_payment_method" || status === "requires_source_action" ) {
```

Comment:
the status we get when setting `payment_behavior: allow_incomplete` is now `requires_source_action`